modules/040-node-manager/docs/FAQ_RU.md

```diff
@@ -1245,3 +1020,333 @@ metadata:
 Некоторые операции по изменению конфигурации узлов могут потребовать перезагрузки.
 
 Перезагрузка узла может потребоваться при изменении некоторых настроек sysctl, например, при изменении параметра `kernel.yama.ptrace_scope` (изменяется при использовании команды `astra-ptrace-lock enable/disable` в Astra Linux).
+
+## Как работать с GPU-узлами?
+
+{% alert level="info" %}
+Функциональность управления GPU-узлами доступна только в редакции **EE**.
+{% endalert %}
+
+### Порядок действий по добавлению GPU-узла в кластер
+
+<span id="порядок-действий-по-добавлению-gpu-узла-в-кластер"></span>
+
+Начиная с Deckhouse 1.71, если в `NodeGroup` есть секция `spec.gpu`, модуль `node-manager` **автоматически**:
```

Comment:
let to add that need use gpu.sharing parameter because default behavior can be changed in the future